toolkit/tools/imagegen/diskutils/diskutils.go

```diff
@@ -660,3 +704,44 @@ func getPartUUID(device string) (uuid string, err error) {
 	uuid = strings.TrimSpace(stdout)
 	return
 }
+
+func getSectorSize(diskDevPath string) (sectorSize uint64, err error) {
+	// Grab the specific disk name from /dev/xxx
+	diskName := diskDevPath[5:len(diskDevPath)]
+
+	hw_sector_size_file := fmt.Sprintf("/sys/block/%s/queue/hw_sector_size", diskName)
+	if exists, ferr := file.PathExists(hw_sector_size_file); ferr != nil {
+		logger.Log.Errorf("Error accessing sector size file %s", hw_sector_size_file)
+		err = ferr
+		return
+	} else if !exists {
+		err = fmt.Errorf("Could not find the hw sector size file %s to obtain the sector size of the system", hw_sector_size_file)
```

Comment:
By Go convention and for consistency let's begin error strings with a small letter.